content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md

```diff
@@ -43,6 +43,12 @@ Additionally you need:
 
 <!-- lessoncontent -->
 
+## Logging
+Generally, application logs are sent to the STDOUT and STDERR streams  and are configured by a logging driver. In the case of Kubernetes, logs are assigned via the kubelet service that runs on the node. You can have a look at the logging architecture(/docs/concepts/cluster-administration/logging) for detailed information 
```

Comment:
The link is not rendering properly. See the preview link provided by @kbhawkey to confirm this.  The syntax should be similar to link to something like:
[PHP Guestbook with Redis](/docs/tutorials/stateless-application/guestbook)